doc/source/getting-started/quickstart.rst

```diff
@@ -0,0 +1,40 @@
+Quick start
+-----------
+
+However, if you are in a hurry, just open a terminal with python > 3.6 and type:
+
+.. code-block:: bash
+
+      pip install qibo
+
+This will install the basic primitives to start coding quantum applications.
+
+
+Here a simple of Quantum Fourier Transform (QFT) to test your installation:
+
+.. code-block:: python
+
+    from qibo.models import QFT
+
+    # Create a QFT circuit with 15 qubits
+    circuit = QFT(15)
+
+    # Simulate final state wavefunction default initial state is |00>
+    final_state = circuit()
+
+Here another example with more gates and shots simulation:
```

Comment:
Since the example that follows does not have more gates than the QFT above, I would probably write
```suggestion
Here an example of adding gates and measurements:
```